src/XamlStyler.Console/Options.cs

```diff
@@ -120,5 +120,8 @@ public sealed partial class CommandLineOptions
 
         [Option("comment-spaces", HelpText = "Override: comment spaces.")]
         public int? CommentSpaces { get; set; }
+
+        [Option("end-of-line", HelpText = "Override: end-of-line.")]
```

Comment:
For consistency with Visual Studio, can we make this option "line-endings" with help text "Override: line endings."

Reply:
@grochocki, you're correct about VS, but most other prominent tools is calling it end of line including VS Code.

- editorconfig: end_of_line
- .gitattributes: [eol](https://git-scm.com/docs/gitattributes#_eol)
- rider: line separators
![image](https://github.com/Xavalon/XamlStyler/assets/6209775/e4f749d5-faf9-43b8-b69d-99a72b7ae04b)
- visual studio code: End of line
![image](https://github.com/Xavalon/XamlStyler/assets/6209775/ca8113d6-5353-4319-88e5-6bd0c8c3dbc1)